Frontend/CO.CDP.OrganisationApp/Session.cs

```diff
@@ -2,38 +2,40 @@
 
 namespace CO.CDP.OrganisationApp
 {
-    public class Session(IHttpContextAccessor httpContextAccessor) : ISession
-    {
-        public T? Get<T>(string key)
-        {
-            var value = GetSession().GetString(key);
-            return value == null ? default : JsonSerializer.Deserialize<T>(value);
-        }
+	public class Session(IHttpContextAccessor httpContextAccessor) : ISession
+	{
+		public const string RegistrationDetailsKey = "RegistrationDetails";
 
-        public void Set<T>(string key, T value)
-        {
-            GetSession().SetString(key, JsonSerializer.Serialize(value));
-        }
+		public T? Get<T>(string key)
+		{
+			var value = GetSession().GetString(key);
+			return value == null ? default : JsonSerializer.Deserialize<T>(value);
+		}
 
-        public void Remove(string key)
-        {
-            GetSession().Remove(key);
-        }
+		public void Set<T>(string key, T value)
```

Comment:
Following [Common c# conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions) we should be using 4 spaces instead of tabs for indentation.